src/main/java/org/spburegistry/backend/service/implementation/ProjectServiceImpl.java

```diff
@@ -96,6 +96,15 @@ public ProjectTO addProject(ProjectTO projectRequest) {
         return ConvertToTO.projectToTO(projectRepo.save(project));
     }
 
+    @Override
+    public Iterable<ProjectTO> findRandomProjects(Optional<Long> limit) {
+        return projectRepo
```

Comment:
Checking limit for negative will be added on https://github.com/spbu-registry/Backend-registry/pull/21. Please, merge that PR firstly